src/actions/users.js

```diff
@@ -559,14 +547,16 @@ export function updateMyTermsOfServiceStatus(termsOfServiceId: string, accepted:
         }));
         const {data, error} = response;
         if (data) {
-            dispatch({
-                type: UserTypes.RECEIVED_TERMS_OF_SERVICE_STATUS,
-                data: {
-                    create_at: new Date().getTime(),
-                    terms_of_service_id: accepted ? termsOfServiceId : null,
-                    user_id: getCurrentUserId(getState()),
-                },
-            });
+            if (accepted) {
```

Comment:
This pattern worries me, because it's a fake `RECEIVED_ME` event. I get that we're just replacing the data we have plus the now known terms of service status, but perhaps it would be better to continue to emit `RECEIVED_TERMS_OF_SERVICE_STATUS` and instead update the user reducer to respond to this event and write the terms of service into the user?

Reply:
Hmm, makes sense. Will be making the changes.

Reply:
Makes sense. Will be making those changes.

Reply:
Done.